docs/Writing_yeb_easyconfig_files.rst

```diff
@@ -116,6 +116,22 @@ elsewhere too) can be done as follows::
 
     versionsuffix: !join [-Python-, *pyver]
 
+.. _easyconfig_yeb_format_syntax_version_majorminor:
+
+Extract major.minor from the version string
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Since it is often needed in easyconfig files to extract the major and minor version number from the version string, the
+EasyBuild framework defines an operator ``!majmin``.
```

Comment:
how about actually calling that `majorminor`? `majmin` is kinda opaque for the unsuspecting bypasser...
I sympathize with the typing-challenged, but this is not all that much found on the command line!